server/game/gamechat.js

```diff
@@ -22,8 +22,8 @@ class GameChat {
         args = _.reduce(args, (argList, arg) => {
             if(arg instanceof Spectator) {
                 argList.push(arg.name);
-            } else if(arg && arg.emailHash) {
-                argList.push({ name: arg.name, emailHash: arg.emailHash, noAvatar: arg.user.settings.disableGravatar });
+            } else if(arg && arg.name) {
```

Comment:
Just so it isn't lost with the other closed out issues - this check is still overly broad and will cause the client to format cards as if they were players:
![screen shot 2018-05-24 at 4 24 09 pm](https://user-images.githubusercontent.com/145077/40518372-27030c7a-5f6f-11e8-8768-c2744441f9d2.png)